.github/workflows/send-email.yml

```diff
@@ -0,0 +1,34 @@
+name: Send mail
+on:
+  workflow_dispatch:
+
+jobs:
+  send-email:
+    runs-on: ubuntu-latest
+    env:
+      FROM_MAIL: julio.ortega@dialpad.com
+      TO_MAIL: brad.paugh@dialpad.com
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Send email
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          secure: true
+          username: ${{ env.FROM_MAIL }}
+          password: ${{ secrets.MAIL_PASSWORD }}
+          subject: Dialtone updates
+          to: ${{ env.TO_MAIL }}
+          from: ${{ env.FROM_MAIL }}
+          html_body: file://apps/dialtone-documentation/docs/about/whats-new/posts/2022-11-18.md
```

Comment:
I wonder if it will handle this md file correctly... hmm guess we'll find out.

Reply:
Yes, it should as it has an option to do it.